HelpSource/Classes/LanguageConfig.schelp

```diff
@@ -52,6 +52,9 @@ Remove path from class library exclude paths.
 METHOD:: currentPath
 Return the current config file path.
 
+METHOD:: excludeDefaultPaths
+Get or set whether default class library paths are included.
```

Comment:
it would be good to note that:
- this value can be overridden if sclang is started with `-a`
- the default class library paths are `Platform.classLibraryDir`, and the system and user extensions directories

Reply:
Added note to the help file.